### ISSUE TITLE
Parentheses

### ISSUE THREAD

**Issue by j-dec** [approach0/search-engine#19]
The search engine can't associate e.g. this:
`\arcsin{\frac{a}{b}}`
with this:
`\arcsin{\left(\frac{a}{b}\right)}`

**Comment by t-k-** [comment]
You are right, but completely ignoring parentheses will result in no difference between `c(x)` and `cx`, the former is a function but the latter is multiplication. I have to admit this is an issue approach0 has to solve in the future. If time permits, I plan to add *query expansion* to (partially) address this issue, for example, to expand query `c(x)` to query expression `cx` OR `c(x)`. But the other way around is similar, the difference is do expansion when indexing document. The draw back of this method is, if there are too many parentheses in a math expression, the time complexity to enumerate all possibilities is huge.
